virttest/env_process.py

```
@@ -665,6 +666,49 @@ def postprocess_vm(test, params, env, name):
+            LOG.warning("VM is not alive so we cannot download extra dump paths")
+        else:
+            session = vm.wait_for_login(timeout=vm.LOGIN_WAIT_TIMEOUT)
+            vm_extra_dumps = os.path.join(test.outputdir, "vm_extra_dumps")
+            if not os.path.exists(vm_extra_dumps):
+                os.makedirs(vm_extra_dumps)
+            for i, dump_path in enumerate(
+                params.get_list("vm_extra_dump_paths", delimiter=";")
+            ):
+                try:
+                    vm.copy_files_from(dump_path, vm_extra_dumps)
+                except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)